crates/bevy_pbr/src/meshlet/cull_meshlets.wgsl

```diff
@@ -21,9 +22,12 @@
 fn cull_meshlets(@builtin(global_invocation_id) thread_id: vec3<u32>) {
     // Fetch the instanced meshlet data
     if thread_id.x >= arrayLength(&meshlet_thread_meshlet_ids) { return; }
+    let instance_id = meshlet_thread_instance_ids[thread_id.x];
+    if (meshlet_view_instance_visibility[u32(instance_id) / 32u] & (1u << (u32(instance_id) & 31u))) != 0u {
```

Comment:
Why are you casting instance_id to a u32? It should already be one. Also can you move this to a seperate function in meshlet_bindings like the others?